apps/server/src/model/cell.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import * as lumos from '@ckb-lumos/base';
+export interface Cell {
```

Comment:
Considering moving to `packages/types`?

Reply:
lumos is used by the server, so it will not be moved to packages/types

Reply:
> Considering moving to `packages/types`?

lumos is used by the server, so it will not be moved to packages/types